client/packages/system/src/Asset/utils.ts

```diff
@@ -1,23 +1,34 @@
 import { LocaleKey, TypedTFunction } from '@common/intl';
 import { AssetCatalogueItemFragment } from './api';
 import { Formatter } from '@common/utils';
+import { ImportRow, LineNumber } from './ImportCatalogueItem';
 
-export const assetCategoryListItemsToCsv = (
-  items: AssetCatalogueItemFragment[],
-  t: TypedTFunction<LocaleKey>
-) => {
-  const fields: string[] = [
-    'id',
+function assetCatalogueItemFields(t: TypedTFunction<LocaleKey>) {
+  return [
+    t('label.sub-catalogue'),
     t('label.code'),
+    t('label.type'),
     t('label.manufacturer'),
     t('label.model'),
+    t('label.class'),
+    t('label.category'),
   ];
+}
+
+export const assetCatalogueItemsListToCsv = (
+  items: AssetCatalogueItemFragment[],
+  t: TypedTFunction<LocaleKey>
+) => {
+  const fields = assetCatalogueItemFields(t);
 
   const data = items.map(node => [
-    node.id,
```

Comment:
Deliberately excluding ID as we don't want it in imports, not sure if adds value in export either except for perhaps our own team?